### PR TITLE
Resolved issue: gatk-launch appends as needed to GATK_GCS_STAGING #1338

### PR DESCRIPTION
**Pull request by MartonKN**
Resolved the issue of adding gs:// to the beginning and / to the end of the environment variable GATK_GCS_STAGING. Tested it locally.

### CODE REVIEW

File: gatk [hunk 1]
```diff
@@ -283,6 +283,13 @@ def md5(file):
 
 def cacheJarOnGCS(jar, dryRun):
     staging = os.environ.get("GATK_GCS_STAGING")
+
+    if staging is not None:
+        staging = staging.strip('/')
+        staging = staging + '/'
+        if not staging[0:5] == "gs://":
```
Comment (davidbenjamin) [inline gatk]: `staging.startswith("gs://")` would be clearer.

**Comment by davidbenjamin** [comment]
Back to @MartonKN.

**Comment by davidbenjamin** [comment]
@MartonKN You can merge once tests pass.

**Comment by MartonKN** [comment]
Closes #1338

**Comment by MartonKN** [comment]
Thank you, @davidbenjamin

**Comment by MartonKN** [comment]
Closes #1338 on Github

**Pull request merged**